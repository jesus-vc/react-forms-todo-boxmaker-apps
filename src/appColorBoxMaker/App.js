import "./App.css";

import BoxList from "./BoxList";

function App() {
  return (
    <div className="ColorBoxApp">
      <BoxList boxCount={20}></BoxList>
    </div>
  );
}

export default App;
