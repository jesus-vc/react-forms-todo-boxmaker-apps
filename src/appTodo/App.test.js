import { render, screen } from "@testing-library/react";
import App from "./App";

// smoke test
it("renders without crashing", () => {
  render(<App />);
});

//snapshot text
it("matches snapshot", () => {
  const { asFragment } = render(<App></App>);
  expect(asFragment()).toMatchSnapshot();
});
