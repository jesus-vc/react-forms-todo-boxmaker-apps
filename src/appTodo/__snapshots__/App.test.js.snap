// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="TodoList"
  >
    <div>
      <ul />
      <div>
        <form>
          <div>
            <label
              for="newTask"
            >
              New Task:
            </label>
            <br />
            <input
              id="newTask"
              name="newTask"
              type="text"
              value=""
            />
          </div>
          <button
            id="newTaskButton"
          >
            Add New Task!
          </button>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
